build.gradle

```diff
@@ -98,6 +98,8 @@ subprojects {
         mavenLocal()
         maven { url "https://aws.oss.sonatype.org/content/repositories/snapshots" }
         mavenCentral()
+        // todo. remove this when lucene 9.4.0 is released
+        maven { url "https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/" }
```

Comment:
why was this added?

Reply:
Good catch! I'll remove it -- it does not apply to 2.x branch.

The upstream PR moved that line because the build was picking up dependencies from that random repository instead of sonatype and failing. 
<img width="689" alt="image" src="https://user-images.githubusercontent.com/99687200/228642982-c39189af-8acf-44ab-8ec1-ec2376196930.png">

It was added when main was [bumped to 3.0](https://github.com/opensearch-project/sql/commit/5dfc2e5ba5d38ce1d0ec35a7904036765e786908).